### ISSUE TITLE
networking issue with load balancers

### ISSUE THREAD

**Issue by zoemccormick** [greymatter-io/terraform-greymatter-ecs#9]
There is some type of networking problem going on with the load balancers in the fabric setup:

Control fails and restarts after trying to connect to control-api with this in the logs:

```

2020-08-18T14:28:51.351-04:00 | <head><title>503 Service Temporarily Unavailable</title></head>
-- | --
  | 2020-08-18T14:28:51.351-04:00 | <body bgcolor="white">
  | 2020-08-18T14:28:51.351-04:00 | <center><h1>503 Service Temporarily Unavailable</h1></center>
  | 2020-08-18T14:28:51.351-04:00 | </body>
  | 2020-08-18T14:28:51.351-04:00 | </html>
  | 2020-08-18T14:28:51.351-04:00 | ', Code: UnknownDecodingCode, Detailed: false}
```

I have also seen it fail with 504s at the same point.

When you ssh into the instance and run `curl localhost:5555/v1.0/zone` you will get a response, but if you try to connect to api using its load balancer dns (as control is), you will see 

```
curl {control-api-dns}:5555/v1.0/zone
<html>
<head><title>503 Service Temporarily Unavailable</title></head>
<body bgcolor="white">
<center><h1>503 Service Temporarily Unavailable</h1></center>
</body>
</html>
```

**Comment by zoemccormick** [comment]
this is in the load balancer monitoring:

<img width="929" alt="Screen Shot 2020-08-18 at 3 22 26 PM" src="https://user-images.githubusercontent.com/32151242/90556332-c1a5be00-e166-11ea-8126-d002d79bb671.png">

**Comment by zoemccormick** [comment]
closed with #8
